packages/blaze/view.js

```diff
@@ -227,7 +227,7 @@ Blaze.View.prototype._errorIfShouldntCallSubscribe = function () {
  */
 Blaze.View.prototype.subscribe = function (args, options) {
   var self = this;
-  options = {} || options;
+  options = options || {};
```

Comment:
It's going to be in Meteor 1.2. Hopefully in the future we can decouple the release process so that we can push fixes in between Meteor releases.